doc/index.md

```diff
@@ -626,6 +626,23 @@ Params:
 
 Returns: a promise with an object with the newly created `conversationId`
 
+#### `startVideoMeeting()`
+Start a video meeting with. Not supported for guests. Meetings can
```

Comment:
I think you meant `Start a video meeting with meetingId`?

Reply:
Seems about right. 🙂